packages/@o3r/eslint-config-otter/package.json

```
@@ -61,15 +61,15 @@
     "@schematics/angular": "~18.2.0",
     "@stylistic/eslint-plugin-ts": "^2.7.0",
     "@typescript-eslint/eslint-plugin": "^7.14.1",
-    "@typescript-eslint/parser": "^7.14.1",
-    "@typescript-eslint/utils": "^7.14.1",
-    "eslint": "^8.57.0",
+    "@typescript-eslint/parser": "^7.14.1 || ^8.0.0",
+    "@typescript-eslint/utils": "^7.14.1 || ^8.0.0",
+    "eslint": "^8.57.0 || <9.15.0",
```

`^8.57.0` is included in `<9.15.0`
```suggestion
    "eslint": ">=8.57.0 <9.15.0",
```